rust/src/http2/http2.rs

```diff
@@ -1201,6 +1217,26 @@ pub unsafe extern "C" fn rs_http2_register_parser() {
         if AppLayerParserConfParserEnabled(ip_proto_str.as_ptr(), parser.name) != 0 {
             let _ = AppLayerRegisterParser(&parser, alproto);
         }
+        if let Some(val) = conf_get("app-layer.protocols.http2.max-streams") {
+            match val.parse::<usize>() {
```

Comment:
I think another `if let` would be more compact and cleaner:
```rust
if let Ok(v) = val.parse::<usize>() {
   ...
} else {
   SCLogError(...)
}
```

Reply:
nice